src/vcastTestInterface.ts

```diff
@@ -653,8 +653,16 @@ function configureWorkspaceAndBuildEnviro(
       .showInformationMessage(message, "Yes", "No")
       .then((answer) => {
         if (answer === "Yes") {
-          fs.mkdirSync(unitTestLocation);
-          commonNewEnvironmentStuff(fileList, unitTestLocation);
+          try {
+            fs.mkdirSync(unitTestLocation, { recursive: true });
+            commonNewEnvironmentStuff(fileList, unitTestLocation);
+          } catch (error) {
+            vscode.window.showErrorMessage(
+              `Error creating directory: ${unitTestLocation}, update the 'Unit Test Location' option to a valid value`
```

Comment:
Is there a chance there's anything in `error` that we should be showing to the user? For example, if it is a permissions problem?

Reply:
Good point, I had not tested a permissions error, dialog looks like this now

![image](https://github.com/vectorgrp/vector-vscode-vcast/assets/5348977/f13b9366-f4c7-4190-9f69-d6e867038d51)

Reply:
Nope